include/net/lwm2m_client_utils.h

```diff
@@ -198,6 +198,117 @@ typedef void (*lwm2m_lte_nwk_reg_update_state_cb_t)(bool connected);
 int lwm2m_lte_reg_handler_register(lwm2m_lte_nwk_reg_update_state_cb_t cb);
 #endif
 
+#if defined(CONFIG_LWM2M_CLIENT_UTILS_GNSS_ASSIST_OBJ_SUPPORT)
```

Comment:
I'm just wondering if it would make sense to split this header file to object specific utility header files?

Reply:
This needs a bit larger surgery. The header-files should be moved to the library level as there should be object specific headers. Also the folder-structure needs changes. But definitely agree, would be a good change.